### DISCUSSION TITLE
[Unstable] Contained item rotation broken

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#15034] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

All contained item sprites no longer rotate with their parent item / follow contained-rotation setting.

### Reproduction steps

![image](https://github.com/user-attachments/assets/7848d047-4f22-4969-bb3a-53e414cbba4e)

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.7.0.1 (Unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Fixed in our private repo now, fix coming up in a future build.
